src/cryptoadvance/specter/server.py

```diff
@@ -62,7 +61,6 @@ def login():
     else:
         app.logger.info("Login enabled")
     app.logger.info("Initializing Controller ...")
-    app.register_blueprint(hwi_views, url_prefix='/hwi')
```

Comment:
The blueprint is a nice feature of flask. Maybe we used it wrong but we should think about how we can leverage it, imho.

Reply:
Blueprint is a nice feature if we need more than one endpoint.
At the moment we use HWI only from javascript and if we make it a single json-rpc endpoint there is no need in a blueprint.
It would make sense to use blueprints for all `wallet/` pages for example, it helps if we have multiple routes with the same prefix.
We should consider blueprints for other things, I just don't think that HWI is the right place to use it.